data/org.freedesktop.portal.RemoteDesktop.xml

```diff
@@ -148,6 +151,13 @@
               A bitmask of the devices selected by the user.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>clipboard_enabled b</term>
+            <listitem><para>
+              A boolean for whether the clipboard was enabled ('true') or not ('false'). 
+              See the #org.freedesktop.portal.Clipboard documentation for more information.
+            </para></listitem>
+          </varlistentry>
```

Comment:
This needs to be specified in `org.freedesktop.impl.portal.RemoteDesktop` as well. Both here and there should this describe what version of `org.freedesktop.impl.portal.RemoteDesktop` it was added.

The `This documentation describes version 1 of this interface.` higher up in this file (and the '.impl' variant) needs the version to be bumped to `2`.